diff --git a/docker/private_docker_lifecycle.go b/docker/private_docker_lifecycle.go
index bb088c6..136dc90 100644
--- a/docker/private_docker_lifecycle.go
+++ b/docker/private_docker_lifecycle.go
@@ -130,7 +130,7 @@ var _ = DockerDescribe("Private Docker Registry Application Lifecycle", func() {
 		It("fails to start the docker app since the credentials are invalid", func() {
 			session := cf.Cf("start", appName)
 			Eventually(session, Config.CfPushTimeoutDuration()).Should(Exit(1))
-			Expect(session).To(gbytes.Say("invalid username/password"))
+			Expect(session).To(gbytes.Say("[Uu]nauthorized"))
 		})
 	})
 })
